unified_planning/model/htn/method.py

```diff
@@ -0,0 +1,235 @@
+# Copyright 2022 AIPlan4EU project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+This module defines the Method class.
+A Method has a name, a list of Parameters, a list of conditions
+and a list of subtasks.
+"""
+from collections import OrderedDict
+from typing import List, Union, Optional
+
+import unified_planning as up
+from unified_planning.environment import get_env, Environment
+from unified_planning.exceptions import UPUnboundedVariablesError, UPValueError
+from unified_planning.model import Timing, Parameter
+from unified_planning.model.action import Action
+from unified_planning.model.htn.task import Task, Subtask
+from unified_planning.model.timing import Timepoint
```

Comment:
Inside a package it is better to do the imports explicitly:
```suggestion
from unified_planning.model.parameter import Parameter
from unified_planning.model.action import Action
from unified_planning.model.htn.task import Task, Subtask
from unified_planning.model.timing import Timing, Timepoint
```

This comment also applies in the files below.